src/main/java/com/accelad/math/nilgiri/DoubleDoubleComplex.java

```diff
@@ -133,8 +131,9 @@ public DoubleDoubleComplex exp() {
     }
 
     public DoubleDoubleComplex log() {
-        return new DoubleDoubleComplex(this.abs().re().log(),
-                new DoubleDoubleReal(FastMath.atan2(imaginary.doubleValue(), real.doubleValue())));
+        DoubleDoubleReal real = this.abs().re().log();
+        DoubleDouble imaginaryPart = DoubleDouble.atan2(this.real.getDoubleDouble(), this.imaginary.getDoubleDouble());
```

Comment:
I thought that:
- If `z = x + iy`
-  `ln(z) = ln(|z|) + i*atan2(y, x)`
Here, I see `Im(ln(z)) = atan2(x, y)` instead of `atan2(y, x)`. Did I misunderstand ?

Reply:
The order of atan2 were inversed -> FIXED